lib/index.js

```diff
@@ -46,187 +43,103 @@ Ramen.prototype.initialize = function() {
  */
 
 Ramen.prototype.loaded = function() {
-  return is.object(window.Ramen);
+  return !!(window._ramen && window._ramen.push !== Array.prototype.push);
 };
 
-/**
- * Identify.
- *
- * @api public
- * @param {Identify} identify
- */
+Ramen.prototype.group = function(group) {
+  if (!this.identified) {
+    this.identify({ anonymous: true });
+  }
 
-Ramen.prototype.identify = function(identify) {
-  privateGo({ integration: this, identify: identify });
-};
+  var props = group.properties();
+  props = alias(props, { createdAt: 'created' });
+  props = alias(props, { created: 'created_at' });
+  var id = group.groupId();
+  if (id) props.id = id;
```

Comment:
Nitpick here but why not `if (group.groupId()) props.id = group.groupId();` ;p if we are not going to use the variable `id` later on in this function